From 15da0fefd64cfeba5ab20d1c531657fc65e2b14d Mon Sep 17 00:00:00 2001
From: slowdown0001 <slow.down.0001.0002.0003@gmail.com>
Date: Sat, 28 Dec 2024 22:06:34 +0900
Subject: [PATCH] include : configs : rpi.h : Add utility command

---
 board/raspberrypi/rpi/rpi.env | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/board/raspberrypi/rpi/rpi.env b/board/raspberrypi/rpi/rpi.env
index 30228285edd..2f2014a9556 100644
--- a/board/raspberrypi/rpi/rpi.env
+++ b/board/raspberrypi/rpi/rpi.env
@@ -75,3 +75,6 @@ fdt_addr_r=0x02600000
 ramdisk_addr_r=0x02700000
 
 boot_targets=mmc usb pxe dhcp
+
+loadenv=fatload mmc 0:1 0x10000000 uboot-env.bin ; env import -c 0x10000000 0x1000
+update_sdcard=run loadenv ; run update_sdcard
-- 
2.34.1

